docs/blog/08-21-2024-introducing-the-federation-api.md

```diff
@@ -0,0 +1,402 @@
+expressive vocabulary for articulating how data may be accessed within the
+ecosystem. Our goal is to improve the user experience of accessing data shared
+by various federation members and to make it easier to build automated data
+access pipelines that seamlessly download the latest pediatric cancer research
+data as it becomes available. Several examples of how to assign and interpret
+these `Gateway`s/`Link`s are available [in the corresponding documentation]
+(**Clay to add link here**).
+
+### Limitations
+
+Being the first version of the API specification, there are several important
+notes to understand when leveraging the API regarding its limitations. We expect
+the list of notes to grow as we receive feedback and questions on this blog
+post.
+
+#### Interpretation of Samples
+
+As discussed in the description of `Sample`s above, this entity covers
+essentially everything between a `Subject` and a `File`. This means that you
+must take great care when interpreting the results from any `/sample` endpoints.
+For example,
+
+- When a single case has been profiled by whole-genome, whole-exome, and
+  RNA-Seq, the server is expected to return three sample entries—not one.
+  Importantly though, the identifier for the sample should be made unique
+  between the three entries when also considering the experiment in the
+  identifier.
+- When the same tumor-banked biospecimen is sequenced by two independent
+  researchers working on different research projects and data from both projects
+  is included in the source server, that server is expected to return two sample
+  entries in that instance—not one. Importantly though, the identifier for the
+  sample should be made unique between the two entries when also considering the
+  namespace in the identifier (different projects should have different
+  namespaces).
+
+This also means that simply counting the number of _cases_ of a particular
+diagnosis is not as simple as using the `GET /sample/by/diagnosis/count`
+endpoint. When counting samples within a particular source server, you should
+consider deduplicating samples across namespaces (if you don't want the same
+biological entity sequenced across multiple projects to count multiple times)
+and/or deduplicating by experiment (if you don't want multiple experiments being
+performed against the same biological entity to be counted multiple times). Even
+in doing so, a comprehensive deduplication of cases across the ecosystem is not
+currently possible for a variety of other reasons, including the fact that no
+facility exists to deduplicate samples across source servers.
+
+### Summary
+
+With these examples laid out in this blog post, we hope we've convinced you that
+a real-time, federated data ecosystem can dramatically improve data discovery
+time (and, subsequentially, time to knowledge generation). To reiterate, all of
+the data data displayed in this blog post was collected from the source servers
```

Comment:
```suggestion
the data displayed in this blog post were collected from the source servers
```

Reply:
See comment about the singular reference to data above.